app/src/main/java/in/ac/iitb/gymkhana/tumtumtracker/MapsActivity.java

```diff
@@ -10,7 +20,9 @@
 import com.google.android.gms.maps.model.LatLng;
 import com.google.android.gms.maps.model.MarkerOptions;
 
-public class MapsActivity extends FragmentActivity implements OnMapReadyCallback {
+import static android.support.v4.content.PermissionChecker.PERMISSION_GRANTED;
```

Comment:
Thats a static int constant  to check if permission was granted or not used in onPermissionRequestCallBack() method